src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -39,5 +39,11 @@
     //  "BlobName": "", // Optional, defaults to Sites/tenant_name/DataProtectionKeys.xml. Templatable, refer docs.
     //  "CreateContainer": true // Creates the container during app startup if it does not already exist.
     //},
+    // Uncomment to configure content localization.
+    //"OrchardCore_ContentLocalization": {
```

Comment:
this should be a whole section `"OrchardCore_ContentLocalization_CulturePickerOptions"` not nested

Reply:
Disagree ;) coz this will introduce many sections per module, Am I right?

Reply:
Yes, but this is a feature setting, for a feature, so it would be problematic if you ever had to add an options configuration for `OrchardCore.ContentLocalization` that was unrelated to the culture picker feature.

One section per feature

Reply:
OK